pkgs/development/libraries/ncurses/default.nix

```diff
@@ -14,17 +14,17 @@
 
 stdenv.mkDerivation rec {
   # Note the revision needs to be adjusted.
-  version = "6.1-20190112";
+  version = "6.2";
   name = "ncurses-${version}" + lib.optionalString (abiVersion == "5") "-abi5-compat";
 
   # We cannot use fetchFromGitHub (which calls fetchzip)
   # because we need to be able to use fetchurlBoot.
   src = let
     # Note the version needs to be adjusted.
-    rev = "acb4184f8f69fddd052a3daa8c8675f4bf8ce369";
+    rev = "v${version}";
   in fetchurl {
     url = "https://github.com/mirror/ncurses/archive/${rev}.tar.gz";
```

Comment:
What do you think about fetching this directly from `https://ftp.gnu.org/gnu/ncurses/ncurses-6.2.tar.gz` instead of third party mirror?

Reply:
At least development versions got removed often, see 43e867a226a8dc5c9c968aeae1c526697c71cd32.